### ISSUE TITLE
ARIA Labels missing from components

### ISSUE THREAD

**Issue by drewhoener** [drewhoener/320-mockup#8]
Some of the components in the project are missing relevant ARIA labels to make the page more accessible to those with screenreaders. Before the project is complete we should go over all the components and make sure that everything has a descriptor
